### PR TITLE
Version 1

### PR DESCRIPTION
**Pull request by testsyren**

### CODE REVIEW

**Comment by testsyren** [comment]
for review

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
+-*~
+-*.tmproj
+-./setup/setenv.sh
+-./sample-proxies/oauth-advanced/oauth2/apiproxy/resources/jsc/build_login_url.js
+-./sample-proxies/oauth-advanced/webserver-app/apiproxy/policies/SetConfigurationVariables.xml
+-./sample-proxies/oauth-advanced/webserver-app/apiproxy/policies/HTMLIndex.xml
+-./sample-proxies/oauth-advanced/login-app/apiproxy/resources/node/config/config.js
+
```
Comment (testsyren1) [inline .gitignore:8]: sample pull comment

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+      "Source" form shall mean the preferred form for making modifications,
+      including but not limited to software source code, documentation
+      source, and configuration files.
+
+      "Object" form shall mean any form resulting from mechanical
+      transformation or translation of a Source form, including but
+      not limited to compiled object code, generated documentation,
+      and conversions to other media types.
+
+      "Work" shall mean the work of authorship, whether in Source or
```
Comment (testsyren1) [inline LICENSE:35]: comment 2

File: default-proxies/helloworld/apiproxy/policies/check-quota.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <Interval ref="request.header.quota_count">1</Interval>
+    <Distributed>false</Distributed>
+    <Synchronous>false</Synchronous>
+    <TimeUnit ref="request.header.quota_timeout">minute</TimeUnit>
+    <StartTime>2016-3-31 00:00:00</StartTime>
+    <AsynchronousConfiguration>
+        <SyncIntervalInSeconds>20</SyncIntervalInSeconds>
+        <SyncMessageCount>5</SyncMessageCount>
+    </AsynchronousConfiguration>
+</Quota>
```
Comment (testsyren1) [inline default-proxies/helloworld/apiproxy/policies/check-quota.xml:15]: no newline at end of file

**Review by testsyren1 — APPROVED** [review]

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,16 @@
+-*~
+-*.tmproj
+-./setup/setenv.sh
```
Comment (testsyren) [inline .gitignore:3]: sample comment

**Pull request merged**